src/modules/item/sagas.ts

```diff
@@ -356,14 +356,18 @@ export function* itemSaga(legacyBuilder: LegacyBuilderAPI, builder: BuilderClien
   function* handleSaveItemSuccess(action: SaveItemSuccessAction) {
     const openModals: ModalState = yield select(getOpenModals)
     const location: ReturnType<typeof getLocation> = yield select(getLocation)
+    const { item } = action.payload
     if (openModals['EditItemURNModal']) {
       yield put(closeModal('EditItemURNModal'))
-    } else if (openModals['CreateSingleItemModal'] && location.pathname === locations.collections()) {
+    } else if (openModals['EditPriceAndBeneficiaryModal']) {
+      yield put(closeModal('EditPriceAndBeneficiaryModal'))
+    } else if (openModals['CreateSingleItemModal'] && location.pathname === locations.collections() && item.type === ItemType.WEARABLE) {
       // Redirect to the newly created item detail
-      const { item } = action.payload
       yield put(push(locations.itemDetail(item.id)))
+    } else if (openModals['CreateSingleItemModal'] && location.pathname === locations.collections() && item.type === ItemType.EMOTE) {
```

Comment:
Question: Should we add the emote's feature flag here as well?

Reply:
Good catch. We should add the emote's FF here.

Doing...